src/thenCatchP.js

```diff
@@ -0,0 +1,29 @@
+import { invoker } from 'ramda';
+
+/**
+ * Composable shortcut for `Promise.then` that allows for success and failure call backs.
+ * The thenCatchP function returns a Promise. It takes three arguments: a callback function for the success of the Promise,
+ * a callback function for the failure of the Promise, and the promise instance itself.
+ *
+ * @func thenP
+ * @memberOf RA
+ * @aliases then
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.8.0|v2.8.0}
+ * @category Function
+ * @sig (a -> Error b -> Promise c | c) -> Promise c | Error b
```

Comment:
```suggestion
 * @sig (a -> b) -> (c -> d) -> Promise a -> Promise b | d
```

Reply:
Thanks for updating the sig! I forgot to check that.